root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -439,6 +439,13 @@ const testData = [
             expected_clean_url: 'https://music.amazon.com/artists/B000QKDMIG',
        only_valid_entity_types: ['artist'],
   },
+  {
+                     input_url: 'https://music.amazon.com/tracks/B0D3QS9NCT#',
```

Comment:
Any reason we only test for removing `#` and not query parameters?

Reply:
Not really. I'll make the change.